pyproject.toml

```
@@ -18,7 +18,10 @@ maintainers = [
     { name = "Juan Nunez-Iglesias", email = "juan.nunez-iglesias@monash.edu" },
     { name = "Martin Durant", email = "mdurant@anaconda.com" },
     { name = "Norman Rzepka" },
-    { name = "Ryan Abernathey" }
+    { name = "Ryan Abernathey" },
+    { name = "David Stansby", email = "d.stansby@ucl.ac.uk" },
```

```suggestion
    { name = "David Stansby" },
```
If it's okay I'd rather not have my email here? Someone can always look it up from my git commits